build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'maven'
 apply plugin: "maven-publish"
```

Comment:
similar cpmments as in https://github.com/splitio/android-client/pull/37

Reply:
I've applied all changes requested in PR#37 (when necessary)